.github/release-drafter.yml

```
@@ -27,6 +27,19 @@ exclude-contributors:
 category-template: '### $TITLE'
 change-template: '* $TITLE ([#$NUMBER]($URL))'
 sort-by: 'title'
+replacers:
+  - search: '/@(\w+)?/g'
+    replace: '[@$1](https://github.com/$1)'
+  - search: '@maxrjones'
+    replace: 'Max Jones'
+  - search: '@michaelgrund'
+    replace: 'Michael Grund'
+  - search: '@seisman'
+    replace: 'Dongdong Tian'
+  - search: '@weiji14'
+    replace: 'Wei Ji'
+  - search: '@yvonnefroehlich'
+    replace: 'Yvonne Fröhlich'
```

Maybe we should include @willschlitzer in this list?

```suggestion
  - search: '@weiji14'
    replace: 'Wei Ji'
  - search: '@willschlitzer'
    replace: 'Will Schlitzer'
  - search: '@yvonnefroehlich'
    replace: 'Yvonne Fröhlich'
```

---

Of course. Thanks for catching it.